### DISCUSSION TITLE
Converting to CSS/SCSS modules from zeit legacy packages

### DISCUSSION THREAD

**Discussion by scottodea** [vercel/next.js#26666] [Help]
Hi, I've been trying to convert to the new out of the box configuration for CSS/SCSS support from the more recent next versions from the previous @zeit/next-css and @zeit/next-sass.
  The issue I'm facing is each component I have in my code base has it's own scss file it's importing. To change each one to a `.module` extension and also change all the classNames from a string to the `styles.someClass` format is just too huge of a project to take on. Is there any alternative to this?  I've tried implementing the `babel-plugin-react-css-modules` plugin with little success. Could anyone provide a setup for this plugin or perhaps preset any alternative solutions? It would be highly appreciated. 
  Thanks

**Comment by morganfeeney** [comment]
Its a really good question, I've just completed this migration where I work. We have roughly 100 component files including templates, it took a team of 3-4 roughly 5 days to complete, this was whilst we had 2 other teams working concurrently, and we had to take a couple of big merges along the way, then do a full regression, it was a big undertaking. It's also hard to justify this kind of work when no new features are being delivered from the developers.

We bit the bullet as next-css becoming deprecated has been on the cards for over a year. Ultimately it seemed inevitable, and not something that could be gradually rolled out.
